libcontainer/cgroups/devices/systemd.go

```diff
@@ -126,7 +140,11 @@ func systemdProperties(r *configs.Resources, sdVer int) ([]systemdDbus.Property,
 				}
 				if group == "" {
 					// Couldn't find a group.
-					logrus.Warnf("could not find device group for '%v/%d' in /proc/devices -- temporarily ignoring rule: %v", rule.Type, rule.Major, *rule)
+					err = fmt.Errorf("systemd older than v240 does not support wildcard-minor rules for devices not listed in /proc/devices: +%v", *rule)
+					if cgroupVer == 2 {
+						return nil, err
+					}
```

Comment:
Surely we don't want to always return an error in this case, even if systemd supports it? I guess it's more consistent but now we're going to start returning errors...